packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -365,4 +400,23 @@ export class QueryBuilderResultState {
       this.isGeneratingPlan = false;
     }
   }
+
+  *getSql(): GeneratorFn<void> {
```

Comment:
could we rename it to generateSql (consistent to generatePlan)? just personal preference